docker-compose.dev.yml

```diff
@@ -15,42 +15,34 @@ services:
         container_name: posthog_redis
         ports:
             - '6379:6379'
-    web:
-        build:
-            context: .
-            dockerfile: dev.Dockerfile
-        command: ./bin/docker-dev-web
-        volumes:
-            - .:/code
-        ports:
-            - '8000:8000'
-            - '8234:8234'
-        environment:
-            DATABASE_URL: 'postgres://posthog:posthog@db:5432/posthog'
-            REDIS_URL: 'redis://redis:6379/'
-            SECRET_KEY: '<randomly generated secret key>'
-            DEBUG: 'true'
-        depends_on:
-            - db
-            - redis
-        links:
-            - db:db
-            - redis:redis
-    worker:
+
+    worker: &worker
         build:
             context: .
             dockerfile: dev.Dockerfile
         command: ./bin/docker-worker
         volumes:
-            - .:/code
+            - .:/code:cached
```

Comment:
What does `:cached` do here?

Reply:
https://www.docker.com/blog/user-guided-caching-in-docker-for-mac/

In the past I have relied on this flag to improve filesystem performance with docker for mac.

However it seems like **I don't need this** because I have the `gRPC FUSE` setting enabled: docker/for-mac#5402

I think it could be a valuable default/flag for "docker-for-mac with osxfs" developers, as long at it doesn't cause issues for others. WDYT?

Reply:
Hm, I think we should avoid it if it's obsolete

Reply:
👍